tests/integrations/raftstore/test_split_region.rs

```diff
@@ -137,7 +137,7 @@ fn test_server_split_region_twice() {
     cluster.must_put(split_key, b"v2");
 
     let (tx1, rx1) = channel();
-    let c = Box::new(move |write_resp: WriteResponse| {
+    let c = Box::new(move |write_resp: Box<WriteResponse>| {
```

Comment:
I doubt if allocation is better than memory copy. And to make it worse, initializing a Box can also be a memory copy, see #4806.

Reply:
@BusyJay OK, thanks for the feedback! I can try implementing @brson's suggestion for using references instead, although I can't do it quickly since I'm on vacationing and not spending much time on the computer.